src/server/session.ts

```diff
@@ -541,31 +556,29 @@ module ts.server {
                             var editorOptions: ts.EditorOptions = {

```

Comment:
do we really need to copy formatOptions? it should be assignable to EditorOptions so original formatOptions can be used directly (assuming that nobody changes them)